configs/mri-recon/mridata-3dfse-knee/unet/vortex/VORTEX_Multi_Aug_UNET_14_Scan.yaml

```diff
@@ -0,0 +1,64 @@
+# Inherit fields from the unet.yaml config.
+_BASE_: "../../templates/unet_multi_aug.yaml"
```

Comment:
Does the `unet_multi_aug` config have augmentations specified in `AUG_TRAIN`? If so, that is not the config you want to inherit from because you don't want their to be augmentation applied in the transform, only in the VORTEX module